### ISSUE TITLE
not launching

### ISSUE THREAD

**Issue by de-tocqueville** [cloveranon/Clover-Edition#206]
So i had it working a few weeks ago, but now when i click play.bat the screen blinks quickly and then nothing. Running windows 10 antivirus is paused, Tried a complete reinstall but same issue. any advice.

**Comment by Engen03** [comment]
Yes, I have the exact same thing. There's one single line in the “launch.py” file: “import aidungeon”, could that be the problem?
